### PR TITLE
doc: add 2016-10-05 diag WG meeting notes

### LINKED ISSUE #64: Diag WG Meeting - 2016-10-05
## Time

05-Oct-2016 Wed UTC 19:00 (1 hour before CTC meeting)

**Please add a 👍  to this post to indicate you'll be attending. Thanks!**

[Click][] for your local time.

[Click]: http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Diag+WG+Meeting&iso=20161005T19

## Join/Watch

If you are primarily observing please use the YouTube channel; Hangouts limits the number of connections. You can interact from YouTube via chat.

* YouTube: _recording was lost_ ☹️ 
* Hangouts: https://hangouts.google.com/hangouts/_/ytl/_xkRfH_3vbTU1gxuNLwhk2aALnssjgvL5h3Er2wq_fo

## Minutes

- Minutes: <https://docs.google.com/document/d/1ksAjU0ypopwZ70kzMOq7XSHrIh20wsOlcFicGbkNly8>
- Previous minutes: <https://docs.google.com/document/d/1rq2VXKb4gj4RBTI1ZfAhRXmMDgO7LDCcLiDeI0FlErI>

---

### v8_inspector/chrome debugging protocol:

- Debug already running process [[node#8464](https://github.com/nodejs/node/issues/8464)]
- Message text for `--inspect` [[node#7182](https://github.com/nodejs/node/pull/7182)]
- Profile for Node? [[diagnostics#52](https://github.com/nodejs/diagnostics/issues/52)]
- V8 inspector with other runtimes [see [node#7393](https://github.com/nodejs/node/issues/7393)]

### V8 TraceController

- To be available in V8 5.4/Node v7.x [[node#8317_comment](https://github.com/nodejs/node/pull/8317#issuecomment-247375183)], [[diagnostics#30](https://github.com/nodejs/diagnostics/issues/30)]
- API and output format [[diagnostics#53](https://github.com/nodejs/diagnostics/issues/53)]
- Feedback on current work
- Next steps

### ES modules

- patching modules after import not possible - how will APMs deal with this?

---

/cc @nodejs/diagnostics

### PR DESCRIPTION
**Pull request by joshgav**
Adds meeting notes, closes #64.

/cc @nodejs/diagnostics

### CODE REVIEW

File: wg-meetings/2016-10-05.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+# Agenda
+
+## Previous Meeting
+
+### async_hooks
+
+Next steps:
+* One significant change remains to enable multiple listeners.
+* @trevnorris is working on a JS wrapper module for AsyncWrap AsyncHooks to make it more usable for end users.
```
Comment (AndreasMadsen) [inline wg-meetings/2016-10-05.md]: ~~AsyncWrap~~ AsyncHooks

File: wg-meetings/2016-10-05.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+
+# Agenda
+
+## Previous Meeting
+
+### async_hooks
+
+Next steps:
+* One significant change remains to enable multiple listeners.
```
Comment (AndreasMadsen) [inline wg-meetings/2016-10-05.md]: Unless this is the wrapper, I'm not sure what it refers too?

Comment (joshgav) [inline wg-meetings/2016-10-05.md]: Oops, actually these lines come from "Next Steps" in https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2016-07-13.md so we better leave them in. Okay?

**Comment by joshgav** [comment]
Updated. @AndreasMadsen PTAL.

@nodejs/diagnostics - can I get another LGTM before merging? Thanks!

**Comment by AndreasMadsen** [comment]
> trevnorris is working on a JS wrapper module for async_hooks to make it more useable for end users.

Its fine. But just to clarify: `async_hooks` is the name of the wrapper for `async_wrap`. It has another name because `async_wrap` is the name of of the C++ class that `async_hooks` (the module API) uses.

**Comment by joshgav** [comment]
@AndreasMadsen 

> edit: though I will say that having the headline "Next steps:" is rather confusing, as it is actually what is happening now. In the previous meeting it made sense, but since then we have moved on.

This part of the notes is actually meant to recap the previous meeting, so I think it's okay that it reflects an old state. More important will be to document this all properly at some point.

**Comment by joshgav** [comment]
landed in 8937270c0c6d21be755d29bba39c199e6dd2352c

**Pull request merged**